.travis.yml

```diff
@@ -34,6 +34,8 @@ matrix:
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: '7.3'
       env: SYMFONY=4.4.*
+    - php: '7.3'
+      env: SYMFONY=5.1.*
     - php: '7.3'
       env: SYMFONY='dev-master as 4.4.x-dev'
```

Comment:
Should we update these environment variables from `dev-master as 4.4.x-dev` to `dev-master as 5.1.x-dev`? It should ensure other dependencies are used at the most likely versions to work?